carve/carve.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/ipfs/go-cid"
+	carv1 "github.com/ipld/go-car"
+	carv2 "github.com/ipld/go-car/v2"
+	"github.com/ipld/go-car/v2/index"
+	"github.com/multiformats/go-multicodec"
+	"github.com/multiformats/go-varint"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:  "carve",
```

Comment:
As much as I like the name `carve`, my vote is to have a `cli` module and simply call the tool `car` following the naming of similar formats like `tar`